app/components/autocomplete/at_mention/at_mention.js

```diff
@@ -111,6 +113,16 @@ export default class AtMention extends PureComponent {
                     });
                 }
 
+                if (groups.length) {
```

Comment:
I've set default to be empty array. 
Furthermore, `searchAssociatedGroupsForReferenceLocal` returns an `[]` if no groups exist.
`searchAssociatedGroupsForReferenceLocal` returns `[]` if you don't have permission or if no groups exist in the state (since `getAllGroups` returns [] if undefined). I believe this shouldn't be a problem anymore